MapboxDirections/MBRouteLeg.swift

```diff
@@ -98,6 +134,26 @@ open class RouteLeg: NSObject, NSSecureCoding {
      */
     open let steps: [RouteStep]
     
+    /**
+     An array of distances representing the distances between nodes.
```

Comment:
(This is a little awkward, because RouteLeg itself doesn’t contain the geometry; only Route and RouteStep do.)